.eslintrc

```diff
@@ -30,7 +30,8 @@
     "react/jsx-uses-react": "error",
     "react/jsx-uses-vars": "error",
     "react/react-in-jsx-scope": "off",
-    "prettier/prettier": ["error", { "singleQuote": true }]
+    "prettier/prettier": ["error", { "singleQuote": true }],
+    "react/prop-types": "off"
```

Comment:
Leave this change for now but we need to discuss the best approach. We'll probably re add it or define a way to describe the props.